cogctl/cli/upgrade_configuration.py

```diff
@@ -0,0 +1,22 @@
+import click
+
+
+@click.command(name="upgrade-configuration")
+@click.pass_obj
+def upgrade_configuration(state):
```

Comment:
I thought it'd be better to have the user do that explicitly. This command won't be around for long, either.